.github/workflows/java.yml

```diff
@@ -313,6 +322,21 @@ jobs:
                   version: "29.1"
                   repo-token: ${{ secrets.GITHUB_TOKEN }}
 
+            # Ensure Rust is in PATH for container environments
+            - name: Setup Rust Build
+              if: ${{ contains(matrix.host.TARGET, 'musl') }}
+              run: |
+                  echo 'source "$HOME/.cargo/env"' >> ~/.bashrc
+                  source "$HOME/.cargo/env"
```

Comment:
You don't need these 2 lines I think since you update `$PATH` on the next line (if you update it properly)
```suggestion
```